tests/modules/asset_groups/test_models.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+import datetime
+import pathlib
+from unittest import mock
+import uuid
+
+
+def test_asset_group_sightings_jobs(flask_app, db, admin_user, test_root, request):
+    from app.modules.asset_groups.models import AssetGroup, AssetGroupSighting
+
+    transaction_id = str(uuid.uuid4())
+    trans_dir = (
```

Comment:
Any reason you didn't use tus.utils.prep_tus_dir here? It seems to be doing the same thing.

Reply:
No, I just didn't check what's available.  I've been setting the tus stuff manually in tests...  But I'll try to remember to use that next time